test/reducers/lastLesson_spec.js

```diff
@@ -0,0 +1,90 @@
+import {expect} from 'chai';
+import deepFreeze from 'deep-freeze';
+
+import reducer from '../../src/reducers/lastLesson';
+
+describe('lastLesson reducer', () => {
+  describe('SET_LASTLESSON', () => {
+    it('Sets last lesson in state to empty string', () => {
+      const initialState = '';
+      const action = {
+        type: 'SET_LASTLESSON',
+        path: ''
+      };
+
+      deepFreeze(initialState);
+      deepFreeze(action);
+      const nextState = reducer(initialState, action);
+
+      expect(nextState).to.equal(action.path);
+    });
+
+    it('Sets last lesson in state to /scratch/astrokatt/astrokatt', () => {
+      const initialState = '';
+      const action = {
+        type: 'SET_LASTLESSON',
+        path: '/scratch/astrokatt/astrokatt'
+      };
+
+      deepFreeze(initialState);
+      deepFreeze(action);
+      const nextState = reducer(initialState, action);
+
+      expect(nextState).to.equal(action.path);
+    });
+
+    it('Missing type', () => {
+      const initialState = '';
+      const action = {
+        path: '/scratch/astrokatt/astrokatt'
+      };
+
+      deepFreeze(initialState);
+      deepFreeze(action);
+      const nextState = reducer(initialState, action);
+
+      expect(nextState).to.equal(action.path);
+    });
+
+    it('Type unknown', () => {
+      const initialState = '';
+      const action = {
+        type: 'unknown',
+        path: '/scratch/astrokatt/astrokatt'
+      };
+
+      deepFreeze(initialState);
+      deepFreeze(action);
+      const nextState = reducer(initialState, action);
+
+      expect(nextState).to.equal(action.path);
+    });
+
+    it('Missing path', () => {
+      const initialState = '';
+      const action = {
+        type: 'SET_LASTLESSON'
+      };
+
+      deepFreeze(initialState);
+      deepFreeze(action);
+      const nextState = reducer(initialState, action);
+
+      expect(nextState).to.equal(action.path);
```

Comment:
Hva skjer hvis path mangler? Skal det kastes exception som man må sjekke for? Eller skal den bare anta at path er tom? Eller hva?